### ISSUE TITLE
Song selection issue

### ISSUE THREAD

**Issue by DarthObi** [ciderapp/Cider-2#422]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://cider.sh/legal/conduct) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ciderapp/Sabiiro-BugTracker/issues) for a bug report that matches the one I want to file, without success.
- [X] I have read the [troublshooting FAQs](https://cider.gitbook.io/welcome-to-gitbook/support/faqs) and none solved my issue.

### What operating system are you using?

Windows

### Build Details

1 month ago

### Operating System Version

Windows 10

### Describe the Bug

Songs being selected are not playing or playing other songs instead

### Steps to Reproduce

Just pressed on a random song in one of my albums

**Comment by camelfert** [comment]
Having this same issue after the Pre-Release After Dark 10/27 and 2.3.0 build from 10/28. Songs in my playlist either won't play if they're far enough into the playlist or just play a totally random song from the list like it's shuffled

**Comment by Gllacyo** [comment]
I would like to propose a solution, it worked for me, I hope it also works on your cider. So I was experiencing the same thing and than I noticed that it was getting worse the more I scrolled but than I noticed that it started skipping by one more song right after a song blocked in my region (gray song pic) so I tried deleting them all and boom it was fixed. However I noticed that it can also occur when searching for songs and it just starts playing the song right after the one you actually clicked in the results window.
